### PR TITLE
[Reset Password] Manage Reset Password permission

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Code Changes
- **user-add-edit.component.html**: Added checkbox input for toggling permission
- **user-add-edit.component.ts**: update `setRequestPermissions` function to set/update the `manageResetPassword` boolean based on the checkbox state
- **messages.json**: `manageResetPassword` checkbox label added

## Screenshots
2-manage-permission

## Dependency
- [Jslib - Permissions API update](https://github.com/bitwarden/jslib/pull/317)

### CODE REVIEW

File: src/app/organizations/manage/user-add-edit.component.html [hunk 1]
```diff
@@ -255,8 +264,8 @@ <h3 class="mt-4 d-flex">
                     <i class="fa fa-spinner fa-spin" title="{{'loading' | i18n}}" aria-hidden="true"></i>
                     <span>{{'save' | i18n}}</span>
                 </button>
-                <button type="button" class="btn btn-outline-secondary"
-                    data-dismiss="modal">{{'cancel' | i18n}}</button>
+                <button type="button" class="btn btn-outline-secondary" data-dismiss="modal">{{'cancel' |
```
Comment (cscharf) [inline src/app/organizations/manage/user-add-edit.component.html:267]: This formatting change is breaking the line in a bad spot IMHO... it'll "work" but it looks really wonky terminating a line with a pipe in the middle of an angular content directive.

**Review by cscharf — COMMENTED** [review]
Just really minor feedback

File: src/app/organizations/manage/user-add-edit.component.html (hunk 1 above)
Comment (vincentsalucci) [reply src/app/organizations/manage/user-add-edit.component.html:267]: I concur - will update!

Comment (vincentsalucci) [reply src/app/organizations/manage/user-add-edit.component.html]: Latest updates should have this fixed for you @cscharf 

👍

**Review by cscharf — APPROVED** [review]

**Pull request merged**